build_site.py

```diff
@@ -4,16 +4,35 @@
 
 from bft.html.builder import build_site
 
+def copy_with_progress(src, dst, copy_function=shutil.copy2):
+    for root, dirs, files in os.walk(src):
+        relative_path = os.path.relpath(root, src)
+        destination_path = os.path.join(dst, relative_path)
+
+        os.makedirs(destination_path, exist_ok=True)
+
+        for file in files:
+            source_file = os.path.join(root, file)
+            destination_file = os.path.join(destination_path, file)
+            copy_function(source_file, destination_file)
+            print(f"Copying: {source_file} -> {destination_file}")
```

Comment:
Let's use pathlib instead of os.path here and in the rest of this file.

You can use this as reference: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
made the change, thanks for the suggestion!